src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -2865,26 +2865,51 @@ The first mechanism is store file refresher which is introduced in HBase-1.0+. S
 * Disable the replication peer named `region_replica_replication` in the cluster using hbase shell or `Admin` class:
 [source,bourne]
 ----
 	hbase> disable_peer 'region_replica_replication'
 ----
 
+Async WAL Replication and the `hbase:meta` table is a little more involved and gets its own section below; see <<async.wal.replication.meta>>
+
 === Store File TTL
 In both of the write propagation approaches mentioned above, store files of the primary will be opened in secondaries independent of the primary region. So for files that the primary compacted away, the secondaries might still be referring to these files for reading. Both features are using HFileLinks to refer to files, but there is no protection (yet) for guaranteeing that the file will not be deleted prematurely. Thus, as a guard, you should set the configuration property `hbase.master.hfilecleaner.ttl` to a larger value, such as 1 hour to guarantee that you will not receive IOExceptions for requests going to replicas.
 
+[[async.wal.replication.meta]]
 === Region replication for META table’s region
-Currently, Async WAL Replication is not done for the META table’s WAL. The meta table’s secondary replicas still refreshes themselves from the persistent store files. Hence the `hbase.regionserver.meta.storefile.refresh.period` needs to be set to a certain non-zero value for refreshing the meta store files. Note that this configuration is configured differently than
-`hbase.regionserver.storefile.refresh.period`.
+Up until hbase-2.4.0, Async WAL Replication did not work for the META table’s WAL. The meta table’s secondary replicas refreshed themselves from the persistent store files every `hbase.regionserver.meta.storefile.refresh.period`,
+(a non-zero value). Note how the META replication period is distinct from the user-space `hbase.regionserver.storefile.refresh.period` value.
+
+Async WAL replication for META is a new feature in 2.4.0 still under active development. Use with caution.
+Set `hbase.region.replica.replication.catalog.enabled` to enable async WAL Replication for META region replicas.
+Its off by default.
+
+Regards the META replicas count, up to hbase-2.4.0, you would set the special property 'hbase.meta.replica.count'.
+Now you can alter the META table as you would a user-space table (if `hbase.meta.replica.count` is set, it
+will take precedent over what is set for replica count in the META table updating META replica count to
+match).
+
+==== Load Balancing META table load ====
+
+hbase-2.4.0 adds a new client-side `LoadBalance` mode. When enabled client-side, clients will try to read META replicas first before falling back on the primary. Before this,
+the lookup mode -- now named `HedgedRead` -- had clients read the primary and if no response after a configurable amount of time had elapsed, it would start up reads against the replicas.
+The new 'LoadBalance' mode helps alleviate hotspotting on the META table distributing the META read load.
```

Comment:
You give a name to AND describe the old behavior, but you only give a name to the new behavior. Would be a kindness to describe and name both, OR just name them and link off to the documentation on this enum in code.

Reply:
Oh. Yes. Let me fix.